changes-set.txt

```diff
@@ -43,13 +43,13 @@ proposed  syl5ibcom imbitridcom
 proposed  syl5ibr   imbitrrid
 proposed  syl5ibrcom imbitrridcom
 proposed  syl5bb    bitrid      compare to bitri or bitrd
-proposed  syl5rbb   bitridcom
+proposed  syl5rbb   bitr2id     compare to bitr2i or bitr2d
 proposed  syl5bbr   bitr3id     compare to bitr3i or bitr3d
-proposed  syl5rbbr  bitr3idcom
+proposed  syl5rbbr  bitr3di     hypotheses would be switched
 proposed  syl5eq    eqtrid      compare to eqtri or eqtrd
-proposed  syl5req   eqtridcom
+proposed  syl5req   eqtr2id     compare to eqtr2i or eqtr2d
 proposed  syl5eqr   eqtr3id     compare to eqtr3i or eqtr3d
-proposed  syl5reqr  eqtr3idcom
+proposed  syl5reqr  eqtr3di     hypotheses would be switched
```

Comment:
Currently, syl5reqr follows the pattern 12 10 02, which does not have a number yet (No. 5 -> eqtr5id?). What does "hypotheses would be switched" mean? Switching their order would result in exchanging the "i" and "d" at the end ("id" means first hypothesis without antecedent, and second hypothesis with antecedent, doesn't it?).

Reply:
The statement "hypotheses would be switched" means that compared to syl5reqr, the hypotheses would be in a different order:

| ![image](https://github.com/metamath/set.mm/assets/58114641/54ea19a3-406d-4c2b-a292-3e5835420648) | ![image](https://github.com/metamath/set.mm/assets/58114641/eb511380-0c7d-455f-9a7f-5ddc2b0ac50e) |

The name eqtr3di is mostly unrelated to syl5reqr 

syl5reqr: (syl5 --> the B in the second hypothesis gets switched out, the two "r" --> the first hypothesis and the conclusion are commuted)

eqtr3di: (eqtr3 --> the left symbols are the same, di --> the first hypothesis has an antecedent, the second doesn't)

The hypotheses are switched because the alternate suggestion here is `eqtr3idcom`, so commuting gives more consistency.